tools/AutomationTestFramework/RunBooks/LiveComputeTests.ps1

```diff
@@ -0,0 +1,59 @@
+Param(
```

Comment:
Should these be checked in?  These look like the generated runbooks created by the scripts?

Reply:
removed.